src/pages/docs/getting-started/adding-a-catalog-item.js

```diff
@@ -0,0 +1,141 @@
+    setTypeFilter(it.id);
+    setAllExamples(it.frontmatter.examples);
+    setKind(sources.find((item) => item.id === it.id).frontmatter?.examples[0]?.name ?? '');
+    setDisplayedKindChips(sources.find((item) => item.id === it.id).frontmatter.examples);
+  };
+
+  return (
+      <section className='sm:grid sm:grid-cols-2 sm:gap-6'>
+        <div>
+          <div className='mb-6'>
+            {sources.map((it) => {
+              return (
+                <button key={it.id} onClick={() => handleChange(it)}>
+                  <Chip label={it.frontmatter.humanName} isActive={typeFilter === it.id} />
+                </button>
+              );
+            })}
+          </div>
+          <CodeBlock language='html' intro={sources.find((it) => it.id === typeFilter).html} />
+        </div>
+        <div className='grid'>
+          {allExamples && !isEmpty(allExamples) && (
+            <div className='self-start sticky top-0'>
+              <div className='mb-6'>
+                {!isEmpty(allExamples) &&
+                  displayedKindChips.map((k) => {
+                    return (
+                      <button key={k.name} onClick={() => setKind(k.name)}>
+                        <Chip label={k.name} isActive={kindFilter === k.name} />
+                      </button>
+                    );
+                  })}
+              </div>
+              <div className='mb-6'>
+                <CodeBlock
+                  language={allExamples.find((kind) => kind.name === kindFilter).language}
+                  code={allExamples.find((kind) => kind.name === kindFilter).content}
+                />
+              </div>
+            </div>
+          )}
+        </div>
+      </section>
+  );
+};
+
+const AddCatalogItemPage = ({ data, location }) => {
+  const sources = data.allMarkdownRemark.edges.map((edge) => edge.node).sort((item1, item2) => item1.frontmatter.order - item2.frontmatter.order);
+  return (
+    <>
+      <DocsHeader location={location} />
+
+      <SEO
+        title={`Adding a catalog item`}
+        description='This tutorial will guide you through the steps required to connect Roadie to your Data source and import the items into the Roadie catalog.'
+      />
+
+      <main className='md:flex pt-4 md:pt-0'>
+        <Sidebar location={location} />
+        {!isEmpty(sources) && (
+          <article className='px-2 md:px-6 md:pt-7 md:flex-1'>
+            <Headline size='small' className="mb-1 mt-0">
+              Adding a catalog item
+            </Headline>
+            <p className='mb-6 mt-6 text-gray-600'>Roadie allows various ways to import items into its catalog. Generally you will store entity files in a code repository and then provide Roadie with the URL of that file for it to be imported as a catalog entry. This tutorial will guide you through the steps required to connect Roadie to your Data source and import the items into the Roadie catalog.</p>
+            <Sources sources={sources} />
```

Comment:
I think the code examples should disappear below a certain breakpoint to avoid the page getting squashed.

![Screenshot 2024-04-22 at 16 20 32](https://github.com/RoadieHQ/marketing-site/assets/562403/cea230ba-8701-4a08-ad05-e64c42beabf5)

Reply:
Ive done this now. I think.